Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1426,6 +1426,64 @@ void CFrame::ParseHotkeys()
 	{
 		State::Load(g_saveSlot);
 	}
+	if (IsHotkey(HK_TOGGLE_STEREO_SBS))
+	{
+		if (g_Config.iStereoMode != STEREO_SBS)
+		{
+			if (g_Config.sPostProcessingShader == "dubois")
+			{
+				g_Config.sPostProcessingShader = "";
+			}
+			g_Config.iStereoMode = STEREO_SBS;
+		}
+		else
+		{
+			g_Config.iStereoMode = STEREO_OFF;
+		}
+	}
+	if (IsHotkey(HK_TOGGLE_STEREO_TAB))
+	{
+		if (g_Config.iStereoMode != STEREO_TAB)
+		{
+			if (g_Config.sPostProcessingShader == "dubois")
+			{
+				g_Config.sPostProcessingShader = "";
+			}
+			g_Config.iStereoMode = STEREO_TAB;
+		}
+		else
+		{
+			g_Config.iStereoMode = STEREO_OFF;
+		}
+	}
+	if (IsHotkey(HK_TOGGLE_STEREO_ANAGLYPH))
+	{
+		if (g_Config.iStereoMode != STEREO_ANAGLYPH)
+		{
+			g_Config.iStereoMode = STEREO_ANAGLYPH;
+			g_Config.sPostProcessingShader = "dubois";
+		}
+		else
+		{
+			g_Config.iStereoMode = STEREO_OFF;
+			g_Config.sPostProcessingShader = "";
+		}
+	}
+	if (IsHotkey(HK_TOGGLE_STEREO_3DVISION))
+	{
+		if (g_Config.iStereoMode != STEREO_3DVISION)
+		{
+			if (g_Config.sPostProcessingShader == "dubois")
```

Comment:
Same as with SBS and top/bottom, only assuring here that anaglyph is not left on when selecting 3D Vision.